packages/driver/cypress/e2e/cypress/proxy-logging.cy.ts

```diff
@@ -363,35 +362,79 @@ describe('Proxy Logging', () => {
           },
         ))
       })
-    })
-  })
 
-  context('Cypress.ProxyLogging', () => {
-    describe('.logInterception', () => {
-      it('creates a fake log for unmatched requests', () => {
-        const interception = {
-          id: 'request123',
-          request: {
-            url: 'http://foo',
-            method: 'GET',
-            headers: {},
-          },
-        }
+      context('with log prop', () => {
+        it('can hide an intercepted request for an image', () => {
+          const logs: any[] = []
+
+          cy.intercept('**/cypress.png*', { log: false }).as('image')
+          .then(() => {
+            cy.on('log:added', (log) => {
+              if (log.name !== 'request') return
+
+              logs.push(log)
+            })
+
+            const img = new Image()
+
+            img.src = `/fixtures/media/cypress.png?${Date.now()}`
+          })
+          .wait('@image')
+          .then(() => {
+            expect(logs).to.have.length(0)
+          })
+        })
+
+        it('uses the final interceptor to determine if a log should be made', (done) => {
+          const logs: any[] = []
+
+          cy.on('log:added', (log) => {
+            if (log.name !== 'request') return
+
+            logs.push(log)
+          })
+
+          cy.intercept('**/cypress.png?*', { log: true }).as('log-me')
```

Comment:
Could shorten this test slightly by combining the two image fetches into one `.then()`, and wait on both requests with the same `.wait()`.

https://docs.cypress.io/api/commands/wait#Usage

I only recently learned you can `cy.wait(['@alias1', '@alias2'])` on an array of aliases, so spreading that learning around a bit. :)

Reply:
Good tip, I didn't know that... somehow :facepalm: I think I'll keep this as-is though, because it's simpler in my mind to first cleanly assert that `dont-log-me` hasn't logged and then separately assert that `log-me` is logged.